### PR TITLE
[gazebo_yarp_forcetorque] Expose fake temperature measurement 

### PR DESCRIPTION
**Pull request by HosameldinMohamed**
Addresses #654

### CODE REVIEW

**Comment by HosameldinMohamed** [comment]
The device fails when trying to use a `multipleanalogsensorsserver` wrapper.

I'm not sure what's missing..


CC @traversaro

**Comment by HosameldinMohamed** [comment]
> The device fails when trying to use a `multipleanalogsensorsserver` wrapper.
> 
> I'm using a separate wrapper while disabling the implicit wrapper!
> 
> I'm not sure what's missing..
> 
> CC @traversaro

Now it's working as expected.
I forgot to implement one method of `yarp::dev::ITemperatureSensors`

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ The format of this document is based on [Keep a Changelog](https://keepachangelo
 
 ## [4.7.0] - 2023-06-05
 
+### Added
+
+- The `gazebo_yarp_forcetorque` plugin now exposes also the `yarp::dev::ITemperatureSensors` interface, so it can stream fake temperature values (https://github.com/robotology/gazebo-yarp-plugins/pull/656).
```
Comment (traversaro) [inline CHANGELOG.md:12]: Can you move this in the `[Unreleased]`  section? Thanks!

**Review by traversaro — APPROVED** [review]

**Comment by traversaro** [comment]
Just a minor comment.

File: CHANGELOG.md (hunk 1 above)
Comment (HosameldinMohamed) [reply CHANGELOG.md:10]: Oops, my bad!

Fixed in https://github.com/robotology/gazebo-yarp-plugins/pull/656/commits/a56ab095cf2122b9086125e6f14fdd2874b8f000

**Review by traversaro — APPROVED** [review]

**Pull request merged**